src/pages/DownloadMonitor.vue

```diff
@@ -1,55 +1,49 @@
 <template>
-    <div class='columns'>
-        <div class="column non-selectable" :class="navbarClass">
-            <NavigationMenu />
-        </div>
-        <div class="column" :class="contentClass">
-            <Hero title="Downloads" subtitle="Monitor progress of downloads" hero-type="is-info"/>
-            <template v-if="activeDownloads.length === 0">
-                <div class='text-center top'>
-                    <div class="margin-right">
-                        <br/>
-                        <h3 class='title is-4'>You don't have anything downloading.</h3>
-                        <h4 class='subtitle is-5'>
-                            Click <a @click="$router.push({name: 'manager.online'})">here</a> to download something.
-                        </h4>
-                    </div>
+    <div>
+        <Hero title="Downloads" subtitle="Monitor progress of downloads" hero-type="is-info"/>
+        <template v-if="activeDownloads.length === 0">
+            <div class='text-center top'>
+                <div class="margin-right">
+                    <br/>
+                    <h3 class='title is-4'>You don't have anything downloading.</h3>
+                    <h4 class='subtitle is-5'>
+                        Click <a @click="$router.push({name: 'manager.online'})">here</a> to download something.
```

Comment:
Would a router-link not work here?

Reply:
Just an oversight, [fixed](https://github.com/ebkr/r2modmanPlus/pull/901/commits/0bc28bfb0529453cfdb10aefa11108eacf8d0416#diff-82438a5b0dc9239c6bd540566979bbb7f12d6845cd406decec09e0f7fd0ac0ebR10).